molecule/plain-rhel/verify.yml

```diff
@@ -230,7 +230,7 @@
               topics: "test_topic"
           - name: test-connector-2
             config:
-              connector.class: "io.confluent.connect.s3.S3SinkConnector"
+              connector.class: "io.confluent.connect.jdbc.JdbcSinkConnector"
```

Comment:
Hi Sneha, I saw the failure msg in the weekly job logs but couldn't understand why this old connector class threw that error.

Reply:
this is test data actually, 
and in 7.1.x branch `io.confluent.connect.s3.S3SinkConnector` is one of the existing connectors for plain-rhel scenario (if you check plain-rhel/molecule.yml), hence test got failed.

To fix it, i'm using `io.confluent.connect.jdbc.JdbcSinkConnector` class for non existent connector.